controls/def.cf

```diff
@@ -126,6 +126,14 @@ bundle common def
                           not(isvariable("trustkeysfrom"))),
         meta => { "defvar" };
 
+      "cf_serverd_maxconnections"
```

Comment:
I don't know what that bundle would look like, but it sounds like a pretty typical use case. How would a bundle be different from specifying the body control settings as usual?